ignite-base/App/Sagas/OpenScreenSagas.js

```diff
@@ -0,0 +1,12 @@
+import { call } from 'redux-saga/effects'
+import { Actions as NavigationActions, ActionConst } from 'react-native-router-flux'
+import { merge } from 'ramda'
+
+// Process OPEN_SCREEN actions
+export function * openScreen (action) {
+  const {screen, options} = action
+  // Always reset the nav stack when opening a screen by default
+  // You can override the RESET type in the options passed to the OPEN_SCREEN dispatch
+  const mergedOptions = merge({type: ActionConst.RESET}, options)
```

Comment:
Totally thinking out loud, but maybe straight up es6 for this?  I mean, personally, i love ramda, in this case, I think the language expresses this clearer.

I'm totally ok if you don't want to.

```js
const navigationOptions = { type: ActionConst.RESET, ...options }
```